package.json

```
@@ -1,55 +1,60 @@
 {
-    "name": "@auth0/angular-jwt",
-    "version": "5.0.2",
-    "scripts": {
-        "ng": "ng",
-        "start": "ng serve",
-        "build": "ng build angular-jwt --prod",
-        "postbuild": "cp ./README.md ./CHANGELOG.md ./LICENSE ./dist/angular-jwt/",
-        "test": "ng test",
-        "lint": "ng lint",
-        "e2e": "ng e2e",
-        "release": "npm publish dist/angular-jwt/"
-    },
-    "private": false,
-    "dependencies": {
-        "@angular/common": "~10.0.1",
-        "@angular/compiler": "~10.0.1",
-        "@angular/core": "~10.0.1",
-        "@angular/platform-browser": "~10.0.1",
-        "@angular/platform-browser-dynamic": "~10.0.1",
-        "core-js": "^3.2.1",
-        "rxjs": "~6.5.3",
-        "tslib": "^2.0.0",
-        "zone.js": "~0.10.3"
-    },
-    "devDependencies": {
-        "@angular-devkit/build-angular": "~0.1000.0",
-        "@angular-devkit/build-ng-packagr": "~0.1000.0",
-        "@angular/cli": "~10.0.0",
-        "@angular/compiler-cli": "~10.0.1",
-        "@angular/language-service": "~10.0.1",
-        "@types/jasmine": "~3.4.1",
-        "@types/jasminewd2": "~2.0.3",
-        "@types/node": "^12.11.1",
-        "codelyzer": "^5.1.2",
-        "jasmine-core": "~3.5.0",
-        "jasmine-spec-reporter": "~5.0.0",
-        "karma": "~5.0.0",
-        "karma-chrome-launcher": "~3.1.0",
-        "karma-coverage-istanbul-reporter": "~3.0.2",
-        "karma-jasmine": "~3.3.0",
-        "karma-jasmine-html-reporter": "^1.5.0",
-        "ng-packagr": "^10.0.0",
-        "protractor": "~7.0.0",
-        "ts-node": "~8.4.1",
-        "tslint": "~6.1.0",
-        "typescript": "~3.9.5"
-    },
-    "np": {
-        "contents": "dist/angular-jwt"
-    },
-    "publishConfig": {
-        "access": "public"
-    }
+  "name": "@auth0/angular-jwt",
+  "version": "6.0.0",
```

Can you revert the version change please?

```suggestion
  "version": "5.0.2",
```